changelog/unreleased/issue-2184

```diff
@@ -0,0 +1,12 @@
+Enhancement: Add --json support to forget command
+
+The forget command now supports the --json argument, outputting the
+information about what is (or would-be) kept and removed from the
+repository.
+
+The text in the paragraphs is written in past tense. The last section is a list
```

Comment:
Remove this paragraph please ;)

Reply:
Gee I dunno, I kinda like it! Reads like a sudden plot twist to a self-describing existential crisis. 😜

Reply:
Removed.